spec/controller/homepage_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe 'Homepage', type: :request do
+  describe 'GET /index' do
+    it 'returns http success' do
+      get '/users/sign_in'
+      expect(response).to have_http_status(:success)
+    end
+  end
+end
```

Comment:
- [OPTIONAL] As requested by the previous reviewer, Kindly make sure to add more integration tests  ✍🏻